R/utils.R

```diff
@@ -91,7 +92,7 @@ extract_model_param_names_h2o <- function(model_param_names, workflow) {
     purrr::pluck("original")
 }
 
-
+# not used
```

Comment:
I merges the parsnip PR that has that function to generate the lists that can be used with `dplyr::rename()`. You can use that in `rename_grid_h2o()`.